lib/internal/url.js

```diff
@@ -1308,11 +1308,17 @@ function originFor(url, base) {
 }
 
 function domainToASCII(domain) {
+  if (arguments.length < 1)
```

Comment:
That would be against the semantics used by other WHATWG URL interface methods. Why the aversion to `arguments` in this case? `arguments.length` is perfectly fine and optimizable by V8.